pkg/git/git.go

```diff
@@ -459,7 +460,31 @@ func (r *Repo) RevParse(rev string) (string, error) {
 	return ref.String(), nil
 }
 
-// RevParseShort parses a git revision and returns a SHA1 trimmed to the length
+// RevParseTag parses a git revision and returns a SHA1 on success, otherwise an
+// error.
+func (r *Repo) RevParse(rev string) (string, error) {
+	// Try to resolve the rev
+	ref, err := r.inner.ResolveRevision(plumbing.Revision(rev))
+	if err != nil {
+		return "", err
+	}
+
+	return ref.String(), nil
+}
```

Comment:
I'm amazed at how much work you put in to rename the old `git.RevParse()`. I think I would have added the functionality to the same function ;)